ddtrace/tracer/span.go

```diff
@@ -135,6 +143,39 @@ func (s *span) setTagError(value interface{}, debugStack bool) {
 	}
 }
 
+// takeStacktrace takes stacktrace
+// skips the stack frame for this function and for the call to runtime.Caller and to the caller -> setTagError
+// also skips the last frame which is runtime
+func takeStacktrace(n, offset uint) string {
+	var builder strings.Builder
+	pcs := make([]uintptr, n)
+
+	i := 0
+	numFrames := runtime.Callers(2+int(offset), pcs)
```

Comment:
What is `2`? Can we leave a comment to the left of this line explaining?

Reply:
I added a longer comment before this line.

Reply:
You left some extra spaces in there. It should suffice to say something like:
```go
// +2 to exclude runtime.Callers and takeStacktrace
```